src/GitHub.VisualStudio/UI/Views/PullRequestDetailView.xaml

```diff
@@ -293,50 +293,61 @@
+                                          Padding="4"
+                                          ContextMenuService.IsEnabled="False"
+                                          ContextMenuService.Placement="Bottom"
+                                          Click="changesOptionsMenuButton_Click">
+                            <ui:OcticonButton.ContextMenu>
+                                <ContextMenu>
+                                    <MenuItem Command="{Binding ToggleOpenChangedFileAction}">
+                                        <MenuItem.Style>
+                                            <Style TargetType="MenuItem">
+                                                <Style.Triggers>
+                                                    <DataTrigger Binding="{Binding OpenChangedFileAction}" Value="Diff">
+                                                        <Setter Property="Header" Value="{x:Static prop:Resources.OpenFileAsDefaultAction}"/>
+                                                    </DataTrigger>
+                                                    <DataTrigger Binding="{Binding OpenChangedFileAction}" Value="Open">
+                                                        <Setter Property="Header" Value="{x:Static prop:Resources.CompareFileAsDefaultAction}"/>
+                                                    </DataTrigger>
+                                                </Style.Triggers>
+                                            </Style>
+                                        </MenuItem.Style>
+                                    </MenuItem>
+                                    <Separator/>
+                                    <MenuItem Command="{Binding ToggleChangedFilesView}">
+                                        <MenuItem.Style>
+                                            <Style TargetType="MenuItem">
+                                                <Style.Triggers>
+                                                    <DataTrigger Binding="{Binding ChangedFilesViewType}" Value="TreeView">
+                                                        <Setter Property="Header" Value="{x:Static prop:Resources.SwitchToListView}"/>
+                                                    </DataTrigger>
+                                                    <DataTrigger Binding="{Binding ChangedFilesViewType}" Value="ListView">
+                                                        <Setter Property="Header" Value="{x:Static prop:Resources.SwitchToTreeView}"/>
+                                                    </DataTrigger>
+                                                </Style.Triggers>
+                                            </Style>
+                                        </MenuItem.Style>
+                                        <MenuItem.Icon>
+                                            <ui:OcticonPath Icon="list_unordered" Fill="Black" Height="16"/>
```

Comment:
Oh no, false alarm: this icon is in a popup menu, and they don't change color according to theme.